src/test/rustdoc/issue-95873.rs

```diff
@@ -0,0 +1,2 @@
+// @!has issue_95873/index.html '{{root}}'
```

Comment:
Please use `@has` instead (we should really forbid `@!has`):

```suggestion
// @has issue_95873/index.html "//*[@class='item-left import-item']" "pub use ::std as x;"
```